warehouse/models/mart/transit_database/dim_organizations.sql

```diff
@@ -44,6 +55,8 @@ dim_organizations AS (
         ON ntd_to_org.ntd_id = ntd.ntd_id
         AND ntd._is_current
         AND ntd.year = 2021
+    LEFT JOIN bridge_organizations_x_county_geography AS geography_bridge
```

Comment:
(reacting to offline convo) sorry I bet the NTD example was confusing here. We don't join the bridge table back onto the dimension because yes in that case it would cause fanout, that's why it lives in the separate bridge table. So we should just remove these left joins of the bridge tables onto the org and service dimensions